app/javascript/app/components/dropdown/dropdown-component.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { PureComponent } from 'react';
+import { ReactSelectize, SimpleSelect } from 'react-selectize'; // eslint-disable-line
+import PropTypes from 'prop-types';
+import Icon from 'components/icon';
+import { themr } from 'react-css-themr';
+import cx from 'classnames';
+
+import arrow from 'assets/icons/dropdown-arrow.svg';
+
+import theme from 'styles/themes/dropdown/react-selectize.scss';
+import styles from './dropdown-styles.scss';
+
+class Dropdown extends PureComponent {
+  componentDidUpdate() {
+    this.selectorElement.highlightFirstSelectableOption();
+  }
+  render() {
+    const {
+      label,
+      white,
```

Comment:
Please remove all this props white, transparent, plain, ... as we are not going to use them and update the docs. If we want to use variations is better to use themes